src/owncloud/autoconfig.php

```diff
@@ -3,23 +3,26 @@
 $snap_name = getenv('SNAP_NAME');
 $snap_developer = getenv('SNAP_DEVELOPER');
 if ($snap_developer == '') {
-        $snap_developer = getenv('SNAP_ORIGIN');
+	$snap_developer = getenv('SNAP_ORIGIN');
 }
 
 $snap_fullname = $snap_name . '.' . $snap_developer;
 
-$data_path = '/var/lib/snaps/'.$snap_fullname.'/current';
+$data_path = '/var/lib/snaps/' . $snap_fullname . '/current';
 
-$AUTOCONFIG = array(
-'directory' => $data_path.'/owncloud/data',
+$writable = getenv('SNAP_DATA');
+$owncloud_password = trim(file_get_contents($writable . '/mysql/owncloud_password'));
```

Comment:
> I still don't get it :D. The variable is used when setting things up, then the file is destroyed and the variable doesn't exist any more anywhere.

I'm talking about [this](https://github.com/kyrofa/owncloud-snap/blob/master/src/owncloud/setup_owncloud#L12). All it uses the file for is to define that variable. If you remove the variable definition from it, then all you have left is "I'm waiting for this file.... okay move on." Though honestly that's no different than what we do for MySQL only a few lines up. The file isn't actually destroyed, and the variable is defined every time (so I quite like your change). I don't think the file should be destroyed, but the variable is definitely not necessary anymore.